tests/protocol.rs

```diff
@@ -1,578 +1,581 @@
-use garble_lang::{
-    circuit::{Circuit, Gate},
-    compile,
-};
-use polytune::{
-    channel,
-    protocol::{mpc, Error},
-};
+#[cfg(not(feature = "is_sync"))]
```

Comment:
You can place `#![cfg(not(feature = "is_sync"))]` at the top of the file and remove the nested `mod tests`. The `!` causes the cfg to apply to the enclosing scope.

This gets rid of the unneeded indentation.